apps/jetbrains/jetbrains.py

```diff
@@ -148,7 +135,15 @@ def idea_commands(commands):
 class Actions:
     def idea(commands: str):
         """Send a command to Jetbrains product"""
-        idea_commands(commands)
+        command_list = commands.split(",")
+        try:
+            for cmd in command_list:
+                if cmd:
+                    send_idea_command(cmd.strip())
+                    actions.sleep(0.1)
+        except Exception as e:
+            app.notify(e)
+            raise e
```

Comment:
I was going to recommend just `raise`, but it looks like in Python 3 this doesn't actually remove any of the context, waited under Python 2? https://stackoverflow.com/a/45986945/303833